docs/src/components/original-documentation/index.js

```diff
@@ -0,0 +1,17 @@
+// Dependencies
+import {h} from 'preact';
+
+// Class
+export default ({children}) => {
+  let child = children && children[0];
+  let orgDocUrl = `https://github.com/material-components/material-components-web/tree/master/packages/${child}`;
```

Comment:
As said above please fix this to point to material.io

Reply:
i will, poves once more DRYer is better ;-)

Reply:
Yes but sadly we all have very limited time to refactor anything at all times and currently I focus more on `2.0`

Reply:
also links in new docs are not consitent with old links any more. little less DRY. Still i'll fix them.

Reply:
> > Finally found a minor alphabetbug in the list of components.
> 
> @prateekbh what do you think, should we simply sort that in code? To be safe of manual sorting.

Dont think its needed, list is not that dynamic and it would make code less readable/performant. In therory we would like to build the list at compile time from the components ;-)

Reply:
> Dont think its needed, list is not that dynamic and it would make code less readable/performant. In therory we would like to build the list at compile time from the components ;-)

That's when rust's `const` would be nice